proto/teleport/lib/teleterm/v1/app.proto

```diff
@@ -75,4 +75,6 @@ message AWSRole {
   string display = 2;
   // ARN is the full role ARN.
   string arn = 3;
+  // AccountID is the AWS Account ID this role refers to.
+  string account_id = 4;
```

Comment:
I think this code in lib/teleterm must be updated to return that account ID in the returned message.

https://github.com/gravitational/teleport/blob/0f57125417769fe6fcae43e0c817ffb0a2d46e58/lib/teleterm/apiserver/handler/handler_apps.go#L72-L76